provider/equinix/userdata.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2020 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package equinix
+
+import (
+	"github.com/juju/errors"
+	jujuos "github.com/juju/juju/core/os"
+
+	"github.com/juju/juju/cloudconfig/cloudinit"
+	"github.com/juju/juju/cloudconfig/providerinit/renderers"
+)
+
+type EquinixRenderer struct{}
+
+func (EquinixRenderer) Render(cfg cloudinit.CloudConfig, os jujuos.OSType) ([]byte, error) {
+	switch os {
+	case jujuos.Ubuntu, jujuos.CentOS:
+		return renderers.RenderYAML(cfg)
+	default:
```

Comment:
Are there any plans to support windows workloads on equinix metal?

Reply:
I think @thebsdbox is the person who has this answer

Reply:
Windows is a supported platform and it can accept userdata.  
https://github.com/dsieczko/terraform-metal-windows-starter/blob/master/main.tf#L10-L18

What are the requirements for a Windows Juju node?